test/Microsoft.ML.Tests/PermutationFeatureImportanceTests.cs

```diff
@@ -40,14 +40,14 @@ public void TestPfiRegressionOnDenseFeatures()
             // X4Rand: 3
 
             // For the following metrics lower is better, so maximum delta means more important feature, and vice versa
-            Assert.Equal(3, MinDeltaIndex(pfi, m => m.MeanAbsoluteError.Mean));
```

Comment:
>       [](start = 0, length = 6)

one too many

Reply:
i mean new lines

---
In reply to: [263174998](https://github.com/dotnet/machinelearning/pull/2804#discussion_r263174998) [](ancestors = 263174998)